src/main/resources/templates/smallfull/employees.html

```diff
@@ -57,13 +57,13 @@ <h2 class="govuk-heading-s">
           <details class="govuk-details" role="group">
             <summary class="govuk-details__summary" role="button" aria-controls="details-content"
```

Comment:
Does this need to be chnaged to employees-details-content to match?

Reply:
Updated to match